stage-ovos/00-build/files/etc/mycroft/mycroft.conf

```diff
@@ -24,21 +24,47 @@
          "ovos-vad-plugin-webrtcvad": {"vad_mode": 3}
          },
       "retry_mic_init" : false,
-      "wake_word": "hey_mycroft"
+      "wake_word": "hey_mycroft",
+      "wake_word_upload": {
+         "disable": true}
```

Comment:
once this makes it into the default config of ovos-core you can remove this + the hotword section and maybe a few others completely